model/task_queue.go

```diff
@@ -399,7 +401,7 @@ func ClearTaskQueue(distroId string) error {
 	aliasQuery := bson.M{
 		taskQueueDistroKey: distroId,
 	}
-	aliasCount, err := db.Count(TaskAliasQueuesCollection, aliasQuery)
+	aliasCount, err := db.Count(TaskSecondaryQueuesCollection, aliasQuery)
```

Comment:
Change the "alias" variables in `ClearTaskQueue` into "secondary" (+ fix the catcher errors).